.gitattributes

```diff
@@ -6,3 +6,7 @@
 # This is relevant for Windows only, and requires git >= 2.19.2 to work.
 /core/libdeno/* symlink=dir
 /cli/tests/symlink_to_subdir symlink=dir
+
+# Tell github that typescrtipt.js is a vendored file.
+# Doesn't include it in the language statistics.
+/cli/tsc/00_typescript.js linguist-vendored
```

Comment:
Cool. I didn't know about that. Maybe we should add cli/dts also.

Reply:
https://github.com/github/linguist#vendored-code